setup.py

```diff
@@ -14,7 +14,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # http://packaging.python.org/en/latest/tutorial.html#version
-    version='0.10.0',
+    version='0.11.0',
```

Comment:
We should probably update the version number only when we're just about to merge develop to master